test/runtests.py

```diff
@@ -7,43 +7,64 @@
 import sys
 import os
 import subprocess as SP
+import argparse, textwrap
+import xml.etree.ElementTree as ET
+
+parser = argparse.ArgumentParser(
+    description='ChakraCore *nix Test Script',
+    formatter_class=argparse.RawDescriptionHelpFormatter,
+    epilog=textwrap.dedent('''\
```

Comment:
I think you could just define the text as a script-level constant instead of defining it inline at this indentation level and taking a dependency on textwrap.

Side note: I'd prefer one package per import line.